docs/frontend/stylelint-settings.md

```diff
@@ -0,0 +1,44 @@
+# StyleLint Settings
+This document describes style lint setting according linter [stylelint.io](https://stylelint.io/).
+
+## Editor Plugins
```

Comment:
```suggestion
## Editor plugins
```
Is there any rule I am missing? When I look into our docs we use capitalization of each word only in page title.

Reply:
Yes, it was my mistake, I thought, we use it for all titles. Changed